test/unit/README.md

```diff
@@ -0,0 +1,9 @@
+# README
+* Last Edited: 2024/02/03
+
+## Description
+Unit tests for relevant classes. 
+
+## Testing Notes
+* Only classes that expose public methods are unit tested.
```

Comment:
This note is really important, I would maybe go ahead and add it as an extra assumption in main readme's scope section,

Reply:
will do now good buddy

Reply:
i've added this to main readme on separate branch and will push after this PR is closed, put it in the test section tho and listed that we're using JUnit 5.8.1 as test framework (that's the version we use, and the version we're supposed to use from Assignment00 doc).